### ISSUE TITLE
Undefined variable in filter

### ISSUE THREAD

**Issue by glydetech** [heanet/moodle_media#1]
```
PHP Notice:  Undefined variable: result in /var/www/moodle/training/filter/heanetmedia/filter.php on line 132
```

https://github.com/heanet/moodle_media/blob/master/heanetmedia/filter.php#L132

The variable $result only gets defined when the regular expression matches.

**Comment by asiera-dev** [comment]
Thanks for the fix Andrew, I've merged it now.

Brian

**Comment by glydetech** [comment]
The zip files haven't been updated in two years and haven't incorporated the fix for Undefined variable in filter #1.

As the repository directory structure is different than the directory structure needed on a Moodle installation, the zip files are the easiest way to install the plugins.

Can you update the zip files or let me know if I should submit a new merge request with the zip files updated. Andrew

**Comment by asiera-dev** [comment]
Thanks for pointing that out. I've just updated the zip files with the fix.

Geoffrey
